### ISSUE TITLE
Component without  give errors in typescript project

### ISSUE THREAD

**Issue by gino8080** [chrisrzhou/react-globe#9]
as subject, using without props says there are lots of missing required props

using simply as:
```javascript
import React, {   SFC } from "react";
import  ReactGlobe  from "react-globe";
 
const Intro: SFC<any> = (props) => {

  return ( <ReactGlobe /> )		

}

export default Intro;
```

Generate a typescript error:

**Type '{}' is missing the following properties from type 'Props': animations, cameraOptions, focusOptions, globeOptions, and 4 more**

I forked https://github.com/gino8080/react-globe to convert all props to optional

**Comment by chrisrzhou** [comment]
Hi @gino8080, I will be pushing out an updated version soon that will fix up Typescript errors in the next few days! I'm new to Typescript but learned a lot when fixing up `react-wordcloud` package recently.

Stay tuned!

**Comment by gino8080** [comment]
Thank you for your nice component!

i just made a PR #10  for a fast "hot fix" if you want to merge, otherwise just ignore it!


cheers

**Comment by chrisrzhou** [comment]
Hi @gino8080, thank you for your PR!  I worked on a more permanent fix in 3.1.1 ([codesandbox example](https://codesandbox.io/s/react-globe-311-cdp92)) by correctly exposing the various Typescript types.  You can install this with `npm` or `yarn`.  The `*Options` props are now typed correctly with optional fields!  Other types are exported as well.  I got into Typescript recently, so I was slowly ramping up on getting this to work.

Thanks for your patience and let me know if you find any other bugs.  Closing the task and PR for now!
